docker_sigprofiler/.dockerignore

```diff
@@ -0,0 +1,3 @@
+__pycache__/
+sp_demo.egg-info
+
```

Comment:
I think this file is in a wrong place. It should be in root folder.

I think you can also excluding following as well:
```
/LICENCE
/INSTALL
/CHANGES.md
/.gitignore
/.git
/.travis.yml
/env
/dist
```